script.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [OPTIONAL] This wasn't mentioned byt the previous reviewer, but as required by Microverse, you are not meant to use any linters disable rule in your project. The linters are there to help you write industry standard code and thereby shouldn't be disabled. :+1: :100:

Reply:
@V-Blaze   I didn't disable any linters?